lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx

```diff
@@ -71,14 +80,28 @@ class HelpPanel extends React.Component {
           css={`
             direction: ltr;
             min-width: 295px;
-            padding: 90px 20px;
+            padding: 84px 25px;
             padding-bottom: 0px;
             display: inline-block;
           `}
         >
-          <Text heading>We&apos;re here to help</Text>
-          <Spacing bottom={5} />
-          <Text medium>
+          <Text
+            css={`
+              color: #404855;
+              font-weight: 800;
```

Comment:
This is better done as a prop called `extraBold` in similar vein to bold/semiBold etc

Reply:
Yup will start doing that from now on